grails-app/services/au/org/emii/aatams/TagFactoryService.groovy

```diff
@@ -18,6 +18,8 @@ class TagFactoryService {
             log.debug("Tag with serial number " + params.serialNumber + " found.")
         }
 
+        loadDeviceStatus(tag, params)
```

Comment:
In hindsight, I think it would have been better to have `DeviceStatus` as a (hard-coded) enum, rather than a database table.  I don't think the set of possible states has ever changed.